api/envoy/api/v2/route/route.proto

```diff
@@ -903,6 +903,26 @@ message RetryPolicy {
     google.protobuf.Duration max_interval = 2 [(validate.rules).duration = {gt {}}];
   }
 
+  message RetryBudget {
+    // Specifies the limit on concurrent retries as a percentage of the sum of active requests,
+    // active connections, and active pending requests.
```

Comment:
I don't think the sum of active connections makes sense here? It really should just apply to active/pending requests IMO.